build/FrameworkPackage/MakeFrameworkPackage.ps1

```diff
@@ -29,6 +29,90 @@ function Copy-IntoNewDirectory {
     }
 }
 
+function ExecuteMakePri
+{
+    Param($WindowsSdkBinDir, $makePriArgs)
+    $makepriNew = "`"" + (Join-Path $WindowsSdkBinDir "makepri.exe") + "`" $makePriArgs"
+    Write-Host $makepriNew
+    cmd /c $makepriNew
+    if ($LastExitCode -ne 0) { Exit 1 }
+}
+
+# only keep 21h1 themeresources, and set othe themeresources to " "
+# I didn't set it to "" because "" will change the `makepri dump` layout, but " " will not.
```

Comment:
So you don't remove the downlevel entries, you just make them empty?

Reply:
yes. To be exactly, the long Base64Value is set to IA==

```
<NamedResource name="rs2_generic_2dot5.xbf" index="18" uri="ms-resource://Microsoft.UI.Xaml.CBS/Files/Microsoft.UI.Xaml/Themes/rs2_generic_2dot5.xbf">
	<Decision index="1">
		<QualifierSet index="0" />
	</Decision>
	<Candidate type="EmbeddedData">
		<QualifierSet index="0" />
		<Base64Value>IA==</Base64Value>
	</Candidate>
</NamedResource>
```

Reply:
Here is the size comparison:
![image](https://user-images.githubusercontent.com/6290692/109880517-de4da300-7c2b-11eb-9594-b0c1c0de2fe9.png)